.github/pull_request_template.md

```diff
@@ -1,61 +1,15 @@
----
-name: "Bug fix or new feature"
-description: ''
-title: ''
-labels: ''
-assignees: ''
-
----
-
 # Description
```

Comment:
```suggestion
```
Is this line necessary?

Reply:
I like the idea of having some sort of prompt. I'd really like to have it as a form, but this is not available on github